### PR TITLE
Icdc 3487

### PR DESCRIPTION
**Pull request by rana22**

### CODE REVIEW

File: src/pages/Cart/CartStyle.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+export default () => ({
+  header: {
+    color: 'red',
```
Comment (ty-esi) [inline src/pages/Cart/CartStyle.js]: use color code

File: src/pages/Cart/CartStyle.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  logo: {
+    width: '96px',
+  },
+  header: {
+
+  },
+  divider: {
+    height: '3px',
+    background: '#686F7F',
+    marginTop: '-13px',
```
Comment (ty-esi) [inline src/pages/Cart/CartStyle.js]: is negative value required here ?

File: src/pages/Cart/components/readMore.js
Comment (ty-esi) [inline src/pages/Cart/components/readMore.js:1]: Why do we have svg code here ?

File: src/pages/JbrowseDetail/components/JBrowseBtnStyle.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  helpIcon: {
+    zIndex: '600',
+    width: '17px',
+  },
+  jbrowseIcon: {
+    width: '2.25em',
+  },
+  diableLink: {
+    pointerEvents: 'none',
+    marginRight: '-5px',
```
Comment (ty-esi) [inline src/pages/JbrowseDetail/components/JBrowseBtnStyle.js]: same here

**Review by ty-esi — APPROVED** [review]
LGTM

File: src/pages/Cart/CartStyle.js (hunk 1 above)
Comment (rana22) [reply src/pages/Cart/CartStyle.js]: updated - removed

File: src/pages/Cart/CartStyle.js (hunk 2 above)
Comment (rana22) [reply src/pages/Cart/CartStyle.js]: removed negative margin

File: src/pages/JbrowseDetail/components/JBrowseBtnStyle.js (hunk 3 above)
Comment (rana22) [reply src/pages/JbrowseDetail/components/JBrowseBtnStyle.js]: removed negative margin

**Pull request merged**